packages/office-ui-fabric-react/src/components/Persona/index.ts

```diff
@@ -2,4 +2,5 @@ export * from './Persona';
 export * from './Persona.base';
 export * from './Persona.types';
 export * from './PersonaCoin';
+export * from './PersonaCoin.base';
```

Comment:
probably want to add PersonaPresence.base too

Reply:
Left out for now as Persona.types export PersonaPresence enum so the names conflict. What are your thoughts on this:

```tsx
export { PersonaPresence as PersonaPresenceIcon } from './PersonaPresence';
export { PersonaPresenceBase as PersonaPresenceIconBase } from './PersonaPresence.base';
```